install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -93,15 +93,9 @@ will approximately take for CNS volumes to be removed from a CNS cluster.
 [[recommended-memory-requirements-for-cns]]
 ==== Recommended Memory Requirements for Container Native Storage
 
-The recommended memory requirements are 32 GB per {product-title} node hosting
-CNS pods.
-
-[IMPORTANT]
-====
-Follow the
-link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.2/html/container-native_storage_for_openshift_container_platform/ch03s02#idm140543609329568[planning guidelines] when planning hardware for a CNS storage environment to ensure that
-you have enough memory.
-====
+Follow the CNS
```

Comment:
I'd say:
To ensure that you have enough memory, follow the CNS
link:https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/3.2/html/container-native_storage_for_openshift_container_platform/ch03s02#idm140543609329568[Planning Guidelines] documentation when you plan hardware for a CNS storage environment.